js/lib/Country.js

```diff
@@ -108,45 +108,45 @@ var Country = function (options) {
 /**
  * Return an array of the ids for all ISO 3166-1 alpha-2 code that
  * this copy of ilib knows about.
- * 
+ *
  * @static
  * @return {Object} an object of country code that this copy of ilib knows about.
  */
 Country.getAvailableCode = function() {
 	var ret = [],
-		country,
+		code,
 		countries = new ResBundle({
-			name: "ctrynames"
+			name: "ctryreverse"
 		}).getResObj();
-	
-	for (country in countries) {
-		if (country && countries[country]) {
-			ret.push(countries[country]);
+
+	for (code in countries) {
+		if (code && countries[code]) {
+			ret.push(code);
```

Comment:
Much more efficient:

```
return Object.keys(this.codeToCountry);
```

The reason is that Object.keys is implemented natively.

Reply:
I agree.